features/step_definitions/series_steps.rb

```diff
@@ -60,3 +54,76 @@
     click_button("Post Without Preview")
   end
 end
+
+Then /^the work "([^\"]*)" should be part of the "([^\"]*)" series in the database$/ do |work_title, series_title|
+  work = Work.find_by_title(work_title)
+  series = Series.find_by_title(series_title)
+  assert SerialWork.where(series_id: series, work_id: work).exists?
+end
+
+Then /^the work "([^\"]*)" should not be visible on the "([^\"]*)" series page$/ do |work_title, series_title|
+  work = Work.find_by_title(work_title)
+  series = Series.find_by_title(series_title)
+  visit series_path(series)
+  page.should_not have_content(work_title)
+end
+
+Then /^the series "([^\"]*)" should not be visible on the "([^\"]*)" work page$/ do |series_title, work_title|
+  work = Work.find_by_title(work_title)
+  series = Series.find_by_title(series_title)
+  visit work_path(work)
+  page.should_not have_content(series_title)
+end
+
+Then /^the work "([^\"]*)" should be visible on the "([^\"]*)" series page$/ do |work_title, series_title|
+  work = Work.find_by_title(work_title)
+  series = Series.find_by_title(series_title)
+  visit series_path(series)
+  page.should have_content(work_title)
+end
+
+Then /^the series "([^\"]*)" should be visible on the "([^\"]*)" work page$/ do |series_title, work_title|
+  work = Work.find_by_title(work_title)
+  series = Series.find_by_title(series_title)
+  visit work_path(work)
+  page.should have_content(series_title)
+  page.should have_content("Part #{SerialWork.where(series_id: series, work_id: work).first.position}")
+end
+
+Then /^the neighbors of "([^\"]*)" in the "([^\"]*)" series should link over it$/ do |work_title, series_title|
+  work = Work.find_by_title(work_title)
+  series = Series.find_by_title(series_title)
+  position = SerialWork.where(series_id: series, work_id: work).first.position
+  neighbors = SerialWork.where(series_id: series, position: [position-1, position+1])
```

Comment:
Surrounding space missing for operator `-`.<br>Surrounding space missing for operator `+`.